sematic/api/endpoints/resolutions.py

```diff
@@ -146,7 +149,9 @@ def put_resolution_endpoint(user: Optional[User], resolution_id: str) -> flask.R
             # Note: This message can be used to extract information about pipeline
             # status for usage in dashboards. Some users may be leveraging it for
             # such purposes, so think carefully before changing/removing it.
-            was_remote = len(resolution.external_jobs)
+            was_remote = (
+                len(get_jobs_by_run_id(resolution.root_id, kind=JobKind.resolver)) > 0
+            )
```

Comment:
Non-blocking nit: implementing a `count_jobs_by_run_id` to save from querying/loading full objects.

Reply:
done